.travis.yml

```diff
@@ -1,8 +1,4 @@
 language: java
 
 jdk:
-  - oraclejdk8
-  - oraclejdk9
-  - openjdk10
-  - openjdk11
-  - openjdk-ea
+  - openjdk8
```

Comment:
Do we know what is the impact of this change?

Reply:
builds were failing with oracle jdks for a long time.
Updated https://github.com/prestodb/airbase/pull/12 to add in open jdk 10 and 11 as well.
will be removing the change from this PR.